packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js

```diff
@@ -18,7 +19,7 @@ const getRegisterCompleteController = (req, res) => {
 			...viewModel[key],
 			ipRefNo,
 			email,
-			nsipProjectLink: nsipProjectLink(req.session.appData)
+			projectURL: getProjectsURL(caseRef)
```

Comment:
This should use the getProjectsIndexURL function.

```suggestion
			projectsIndexURL: getProjectsIndexURL(caseRef)
```